app/models/intake/ctc_intake.rb

```diff
@@ -440,4 +440,8 @@ def spouse_prior_year_agi_amount_computed
   def puerto_rico_filing?
     home_location_puerto_rico?
   end
+
+  def claiming_eitc?
```

Comment:
YAY. I like this pattern. Can you let me know once this is merged so that Maria and I can change the code we wrote in intake to use this method?